static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -68,16 +68,15 @@ define([
         }
     };
 
-    policies.membershipMessages = function () {
-        if (!detect.adblockInUseSync() &&
-            detect.getBreakpoint() !== 'mobile' &&
-            config.page.contentType === 'Article' &&
-            !userFeatures.isPayingMember()
-        ) {
-            return {
-                membershipMessages : true
-            };
-        }
+    policies.checkWeCanShowMembershipMessages = function () {
```

Comment:
@OliverJAsh Policies return maps that then get squashed together. The idea - back at the time - was that this meant a policy could map an abstract idea to a set of feature changes, e.g.

    policies.nonEnhancedUsers = ()=> {
        if (config.isEnhanced === false) {
            return {
                resizableTopNav : false,
                highPerformanceTopNav : true,
                viewabilityTracking : false
            }
        }
    }

et cetera, et cetera.

@sndrs Is naming things `promise` naming them after their implementation, or their contract / interface? I would say `checkWeCanShowMessages` sounds like a function to me.